webviews/common/context.tsx

```diff
@@ -158,6 +158,7 @@ export class PRContext {
 		});
 		state.reviewers = reviewers;
 		state.events = [...state.events.filter(e => (isReviewEvent(e) ? e.state !== 'PENDING' : e)), review];
+		state.reviewState = review.state;
```

Comment:
Is a check needed here to make sure that the review is from the current user?

Reply:
@alexr00 I did not see a need to check if the review is from the current user as this refers to the review state of the PR irrespective of who did it. It is required to enable/disable the Approve button on the overview of the PR. 
When the PR panel is shown it always refers to the review state from the current user for the PR
When the reviewer makes changes to the PR ```review``` always refers to the current user's review on the PR.
Please let me know if I am missing anything here, I am happy to make changes/updates